contributing/testing.md

```diff
@@ -10,6 +12,24 @@ From the project root run:
 npm run lint
 ```
 
+### Ruby linting
```

Comment:
Thanks for adding this! I have https://github.com/citizensadvice/design-system/pull/3496 open right now which makes some larger changes to these guides. I've tried to incorporate your changes into there. Mind taking a look and if you are happy pulling these changes out of this PR?

Reply:
Sorry just catching up after holidays. #3496 looks good, happy to take these out